cli/tests/testdata/event_listener_error_immediate_exit.ts.out

```diff
@@ -1,4 +1,5 @@
 1
+queueMicrotask
```

Comment:
@nayeemrmn @lucacasonato is this test supposed to be always deterministic? queueMicrotask is called now maybe because we don't enter v8 callconsole?